CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+# CLARIAH Interest Group Contribution Guidelines
+
+1. Each Interest Group has its own **Github repository** with associated **issue tracker**, all interests groups are
+   part of the [CLARIAH organization](https://github.com/CLARIAH) on Github.
+     - There is one [over-arching repository](https://github.com/CLARIAH/IG) that provides [an index to all individual interest groups](https://github.com/CLARIAH/IG/blob/main/README.md)
+       and holds documents that transcend individual interest groups. Such as meeting notes and documents
+       from the Technical Committee.
+     - There is another central repository that holds all [use cases](https://github.com/CLARIAH/usecases) for all
+       Work Packages and Interest Groups.
+2. All relevant documents and other group output should be gathered in the relevant repository.
+    - This ensures everything can be found in one place and is under proper version control and that everybody
+      is free to use the tools they prefer to edit the contents.
+    - This provides transparency, findability, and accountability.
+    - If there are relevant resources on external platforms, such as a document on Google docs for instance,
+      then that is of course fine, but make you **link to it** from one or more documents this repository, simply
+      putting a URL in relevant document will do. Otherwise people will not know about the existence of your document.
+    - If the output of the group consists of custom-made software then that will be hosted in its
+      own separate repository and will be linked to from one of more documents in this repository.
+    - Keep documents in **plain text** as much as possible, or more specifically: use
+      [Markdown](https://guides.github.com/features/mastering-markdown/) syntax and always use the UTF-8 character
+      encoding. Prevent proprietary closed formats and binary formats as much as possible, but if you can't avoid it,
+      just commit them to the repository.
+    - If your group gives presentations, please put the slides in the repository.
+3. Group discussion can take place through the **issue tracker** associated with a repository.
+    - If your group holds meetings (virtual or physical), please ensure to always put the meeting notes in the
+      git repository afterwards. Or, preferably, try to
+      summarize any actionable points in issues on your group's issue tracker.
```

Comment:
Do you also propose to keep the TC tasks on github issues?

Reply:
Yes, I think it would be good to apply the same format to everything. We could use the issue tracker of this generic IG group for the TC.